test_parenthetics.py

```diff
@@ -0,0 +1,20 @@
+"""Test file for parenthetics.py."""
+
+
+import pytest
+
+TEST_LIST = [
+    ['((()))', 0],
+    ['(what((for)jds', 1],
+    ['hi () whst(idf)jkdf0)', -1],
+    [')jfei()nsdf(fnif)j(', -1],
+    ['()dhuew(dai0dasud)dnaudb(fkabfa(fkhaush)', 1],
+    ['adfi()fhsd(ffjausd)fhaeb()hfa9', 0],
```

Comment:
You shouldn't need to be testing with all these various other characters. You should just be testing with different combinations of nested parens